charts/pega/templates/_helpers.tpl

```diff
@@ -236,3 +236,11 @@ true
     # Used to specify permissions on files within the volume.
     defaultMode: 420
 {{- end}}
+
+{{- define "isHzEnabled" }}
+  {{- if (eq .Values.hazelcast.enabled true) -}}
+    true
+  {{- else -}}
+    false
+  {{- end -}}
+{{- end }}
```

Comment:
We already have this template in hazelcast subchart helpers.tpl . Is there any need to have it at this place as well.

Reply:
removed this template and modified the references